packages/sn-filter-pane/src/hooks/use-render.ts

```diff
@@ -1,28 +1,33 @@
-import { useEffect, useElement } from '@nebula.js/stardust';
+import { useEffect, useElement, useState } from '@nebula.js/stardust';
 import type { IStores } from '../store';
 import useListBoxResources from './listbox/get-listbox-resources';
 import { IContainerElement } from './types';
 import { render, teardown } from '../components/root';
 
 export default function useRender(stores: IStores) {
+  const [mounted] = useState<{current: boolean}>({ current: true });
```

Comment:
Unsure if it is more logic to have this variable in `useListBoxResource` or here